packages/flutter/test/material/list_tile_test.dart

```diff
@@ -349,16 +349,20 @@ void main() {
           child: MediaQuery(
             data: const MediaQueryData(),
             child: Column(
-              children: const <Widget>[
-                ListTile(
+              children: <Widget>[
```

Comment:
Does this mean that when a widget is interactive, it's not necessarily focusable? 

I'm thinking about the case where `ListTile.enabled = true` and `onTap`/`onLongPress` is set to null. If I understand this correctly, this might just mean that the ListTile is "active", but not really interactive and is more so an issue with the wording in the API docs.